### ISSUE TITLE
[Docs] Make sure all Colab notebooks work

### ISSUE THREAD

**Issue by zhe-thoughts** [ray-project/ray#33887]
### What happened + What you expected to happen

Some of our examples link to a Colab notebook 
![Screenshot 2023-03-29 at 2 31 41 PM](https://user-images.githubusercontent.com/10713559/228672751-3baf0186-360d-4bc8-928b-9d042da78526.png)

But some Colab notebooks do not work out of the box. E.g. https://docs.ray.io/en/master/ray-air/examples/pytorch_resnet_batch_prediction.html and https://colab.research.google.com/github/ray-project/ray/blob/master/doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

We should systematically make sure all of these notebooks are tested

### Versions / Dependencies

2.3.1 (latest)

### Reproduction script

https://colab.research.google.com/github/ray-project/ray/blob/master/doc/source/ray-air/examples/pytorch_resnet_batch_prediction.ipynb

### Issue Severity

High: It blocks me from completing my task.

**Comment by zhe-thoughts** [comment]
Internal Slack thread: https://anyscaleteam.slack.com/archives/C02UGNK9080/p1680124998810919

**Comment by anyscalesam** [comment]
re-reviewing - this thread died off 7mo ago. @zhe-thoughts is this still high priority currently?

**Comment by angelinalg** [comment]
@can-anyscale, is this something you can help us with?

**Comment by can-anyscale** [comment]
Yes, feel free to re-assign it to me. The priority discussion can come into next sprint + headcount updates.

**Comment by zhe-thoughts** [comment]
Thanks @anyscalesam , @can-anyscale

**Comment by dstrodtman** [comment]
Closing as obsolete. Ray AIR was deprecated in 2023, and the APIs referenced here have moved into Ray Train (typically with reshaped surface area). If you're hitting a similar gap in the current Train docs, please open a fresh issue with a link to the page.

Context: docs-team triage of the OSS docs issue backlog.
